### DISCUSSION TITLE
Permanent skipping of the location marker and moving of the map

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#190] [Q&A] [answered]
Yesterday I changed to my backup phone Samsung A52S 5G and had the same problem that I had once on my Galaxy S9+, but all the time. The location marker skipped wild on the screen with map moving there and back.
My default setting is Kalman filter off and Auto Zoom Mode Speed reverse and distance and animated location on. With animated location on it was the most terrible, but I also changed all the other settings, Kalman filter Zoom modes, location service between Android and Google and the effect didn't stop. I couldn't make the tour and had to change the phone back to my S9+. With this everything works normal.
The only physical difference is that the Silicone protective cover of the A52S is a little thicker than this from the S9+ and has a protective glass instead of protective film.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

Do you mean when navigation was enabled, the location jumped on the screen and the map followed the location?
Were you moving or standing still while this happened? When you weren't moving, what happened?

Indeed the "Animated location" and "Kalman filter" must be switched off and test again.

Also disable the "Settings | Navigation | Advanced | Snap to route".
(which I wonder if we really need)

Have you tested with the "Location service" to Android or Google?

Did you have other apps running while this happened?

Can save your settings and then _"Reset" settings to defaults_ and then test again.
Also a restart of the phone may help.

Also when this happens, can you export the app settings and post the .pref file?

**Comment by DolomitesTom** [comment]
> Do you mean when navigation was enabled, the location jumped on the screen and the map followed the location?
yes exactly

> When you weren't moving, what happened?

Then the location cursor was exactly positioned on the track.


> Also disable the "Settings | Navigation | Advanced | Snap to route".
(which I wonder if we really need)

This was the only feature that stayed active. All others have been changed and then tested again while driving to the meeting point.

> Have you tested with the "Location service" to Android or Google?

Yes, both. No difference.

> Did you have other apps running while this happened?

Yes an other GPS overlay and warning app. But even after closure of this app, the jumps where there, potentially not that intensive any more, but still no clear navigation possible, many jumps.

> Also when this happens, can you export the app settings and post the .pref file?

Next time when driving

**Reply by devemux86** [reply]
> This was the only feature that stayed active.

Please disable "Snap to route" next time (it is off by default).

Were you in an area where the route geometry was overlapping / complicated?
Or it was happening everywhere, even in long straight paths?

If possible, a video would also help. :slightly_smiling_face:

**Reply by DolomitesTom** [reply]
Had no time to record video, I was on the way to the meeting point. Today I made a 150km short trip with my Backup phone and switched off "Snap to route". Everything was fine no jumps anymore.

**Reply by devemux86** [reply]
@DolomitesTom thanks for the tests!

If we can narrow down the cause to the "Snap to route" option, it will help me to better examine why this is happening.
Also, if you can post a specific route that happens, unless it also happens in long straight lines.

**Comment by devemux86** [comment, marked as answer]
Better to remove the "Snap to route" setting for now. It's more for effect than real use.

Other apps that snap user location have serious problems with lags and strange directions. :slightly_smiling_face:
